tests/unit/algo/classification/test_otx_dino_v2.py

```diff
@@ -0,0 +1,32 @@
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+from __future__ import annotations
+
+import torch
+from otx.algo.classification import DINOv2
+
+
+class TestDINOv2:
+    def setup(self) -> None:
+        self.model = DINOv2(
```

Comment:
```suggestion
    @pytest.fixture
    def model(self) -> None:
        return DINOv2(
```

I have not been noticed but I guess there were so many anti-patterns which is far away from the pytest common  practice, not only this PR, but also https://github.com/openvinotoolkit/training_extensions/pull/2711.

Please do not use member variable, use pytest fixture.

Reply:
Thanks, https://github.com/openvinotoolkit/training_extensions/pull/2708/commits/00153bb018437a0ca6120ac610f8aaf3501d9bd9